cmd/util.go

```diff
@@ -57,7 +55,7 @@ func manifestFromYAML(path string) ([]unstructured.Unstructured, error) {
 			if err == io.EOF {
 				break
 			}
-			return nil, fmt.Errorf("could not read YAML: %s into a manifest: %s", path, err)
+			return nil, fmt.Errorf("could not read YAML into a manifest: %s", err)
```

Comment:
Didn't think the path was important here, but open to finding a way to add it back if they did pass a file

Reply:
I think it'd be good to have since the plugin can be operate on all the files in a directory, so if one is malformed it'd be obvious which one

Reply:
Cool I think I can come up with something!